production/helm/loki/templates/backend/statefulset-backend.yaml

```diff
@@ -68,6 +68,73 @@ spec:
         {{- end }}
       {{- end }}
       containers:
+      {{- if .Values.sidecar.rules.enabled }}
```

Comment:
We actually added the sidecar to the `read` statefulset and that works. But we were never sure in what part of loki the rules need to go.
Does this work for you?